assets/src/dashboard/app/views/shared/storyGridView.js

```diff
@@ -73,8 +74,14 @@ const StoryGridView = ({
   dateSettings,
   previewStory,
 }) => {
+  const gridRef = useRef();
+
   return (
-    <StoryGrid pageSize={pageSize}>
+    <StoryGrid
+      pageSize={pageSize}
+      gridRef={gridRef}
+      ariaLabel={__('Grid view of stories', 'web-stories')}
+    >
```

Comment:
minor ground work, rest of this update will come separately. just passing the props the cardGrid now wants

Reply:
Can the grid ref just be created in the `StoryGrid` component?

Reply:
i updated this to forward the ref, per carlos's suggestion -  https://github.com/google/web-stories-wp/pull/4317#discussion_r482616366 

we need the gridRef in the `storyGridView` and `templateGridView` to control which grid item is in focus.